### DISCUSSION TITLE
Where can I find detailed descriptions of reservoir models in GEOPHIRES-X?

### DISCUSSION THREAD

**Discussion by huhanyu** [NatLabRockies/GEOPHIRES-X#145] [Q&A] [answered]
Hello, I am currently writing a research paper on [Your Research Topic Here] using the GEOPHIRES-X project.
Specifically, I am looking for detailed descriptions of models such as the Multiple Parallel Fractures model, the 1D Linear Heat Sweep model, and any other relevant models. Could someone please direct me to where I can find this information? Any documentation, papers, or tutorials related to these models would be greatly appreciated. Thank you in advance for your help!

**Comment by softwareengineerprogrammer** [comment]
@kfbeckers or @malcolm-dsider can either of you give pointers for this?

**Reply by kfbeckers** [reply, marked as answer]
Take a look at section 4.3.1 in https://ecommons.cornell.edu/items/ca2cdbf7-afe3-4df0-98e6-ee274f7fc5e7

**Reply by huhanyu** [reply]
thank you ！
